### PR TITLE
Header feature

### PR DESCRIPTION
**Pull request by peterdgreat**
This is a complete styling of the mobile contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -284,6 +284,42 @@ <h1>
...
+       <form class="form" action="https://formspree.io/f/xeqvpzae" method="post">
+           <ul>
+               <li class="border-bottom">
+                   <input type="text" id="name" placeholder="Full name" required maxlength="30">
+               </li>
+               <li class="border-bottom">
+                   <input type="email" name="user-email" id="email" placeholder="Email address" required>
+               </li>
+               <li>
+                   <textarea name="comment" id="comment" placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (DanteAlonsoHT) [inline index.html]: The message should have `required`  as an attribute according to the project requirements.

- [ ] Add `required` in `textarea` as attribute.

File: index.html [hunk 2]
```diff
@@ -284,6 +284,42 @@ <h1>
...
+   </section>
+   <footer>
+    <hr class="bg-footer">
+ <ul class="footer-social">
+     <li> <a ><img src="./img/f-GitHub.png" alt="github" ></a> </li>
+     <li><a ><img src="./img/f-Linkedin.png" alt="Linkedin" > </a></li>
+     <li><a > <img src="./img/f-Angelist.png" alt="Angelist"></a> </li>
+     <li><a ><img src="./img/f-Twitter.png" alt="twitter"> </a> </li>
+     <li><a ><img src="./img/f-Medium.png" alt="Medium"> </a> </li>
+ </ul>
```
Comment (DanteAlonsoHT) [inline index.html]: For all of your link tags (`a`) you should add the `href=""` attribute, if they don't have an available link, just put the "#" value for `href` attribute, For instance:

```
<a href="#"></a>
```

- [ ] For all of your `a` tag, add `href=""`.

File: style.css [hunk 3]
```diff
@@ -425,3 +476,88 @@ hr {
...
+
+button:enabled {
+  background: #f55800;
+}
+
+button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
```
Comment (DanteAlonsoHT) [inline style.css]: - [ ] [OPTIONAL] Add pointer cursor when you hover a button. For instance: 
```
cursor: pointer;
```

File: index.html [hunk 4]
```diff
@@ -284,6 +284,42 @@ <h1>
 </ul>
     </section>
 </div>
-
+<div>
+   <section class="contact-form">
+       <div class="interest">
+           <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+       </div>
```
Comment (DanteAlonsoHT) [inline index.html]: In order to have better practices with HTML and Improve the SEO for your websites, you must use one `h1` tag per HTML file, for the rest of your header, you should use `hN` where N is a number between (2...6).

- [ ] Use only one `h1` per HTML file.
- [ ] Apply the same described above for your whole website.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @peterdgreat, 

Amazing so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Improve your README file, you can see [This Example](https://github.com/DanteAlonsoHT/OnlineCourse-FInalCapstone) about what we expect from your README files in the whole instance in Microverse. 😉 

_Check the comments under the review._

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

<hr>

I know you have the skills enough to finish these changes required in time 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @peterdgreat  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**